src/Cake.Prca/Issues/ICodeAnalysisIssue.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Cake.Prca.Issues
+{
+    using Core.IO;
+
+    /// <summary>
+    /// Description of a code analysis issue.
+    /// </summary>
+    public interface ICodeAnalysisIssue
+    {
+        /// <summary>
+        /// Gets the path to the file affacted by the issue.
+        /// The path is relative to the repository root.
+        /// </summary>
+        FilePath AffectedFileRelativePath { get; }
+
+        /// <summary>
+        /// Gets the line in the file where the issues has occurred.
+        /// </summary>
+        int Line { get; }
+
+        /// <summary>
+        /// Gets the message of the code analysis issue.
+        /// </summary>
+        string Message { get; }
+
+        /// <summary>
+        /// Gets the priority of the message used to filter out issues if there are more issues than should be posted.
```

Comment:
@stffabi What would you expect? 

Currently issues are ordered ascending by priority and then a `Take()` is made. Thinking about it I would expect it the opposite way (higher priority = more likely to be posted).